starter/src/app/_components/home/filters.tsx

```diff
@@ -0,0 +1,73 @@
+"use client";
+
+/* */
+import Results from "@/app/_components/home/results";
+import SidebarItem from "@/app/_components/home/sidebar-item";
+import SignupCard from "@/app/_components/home/signup-card";
+import { useBreakpoint } from "@/app/_components/universal/hooks/use-breakpoint";
+import { Layout } from "@/app/_components/universal/layout";
+import { sidebarCategories, resultItems } from "@/app/constants";
+import type { SidebarCategory, ResultItem } from "@/app/constants";
+import { Button } from "@/components/ui/button";
+import {
+  Dialog,
+  DialogPortal,
+  DialogOverlay,
+  DialogClose,
+  DialogTrigger,
+  DialogContent,
+} from "@/components/ui/dialog";
+
+export const HomeFilters = () => {
+  const isMd = useBreakpoint("md");
```

Comment:
TODO: isMd needs some time for loading, probably we need a skeleton here while it is loading

Reply:
You can avoid the loading state by including both sidebars into the code and apply `hidden md:block` to the version from `md` upwards.

This is how the dashboard navigation works:
https://github.com/calcom/examples/blob/main/starter/src/app/dashboard/layout.tsx